bin/dev

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def installed?(process)
+  IO.popen "#{process} -v"
+rescue Errno::ENOENT
+  false
+end
+
+def run(process)
+  system "#{process} start -f Procfile.dev"
+rescue Errno::ENOENT
+  warn <<~MSG
+    ERROR:
+    Please ensure `Procfile.dev` exist in your project!
```

Comment:
grammar

`exists`

Reply:
My bad!
This should be fixed in ROR as well. (made pr there)

Should I fix it manually here or wait for another release on ROR and then update them here?